doc/languages-frameworks/haskell.section.md

```diff
@@ -519,7 +571,7 @@ enter a shell with all the package's dependencies available using `nix-shell
 -A env default.nix`. If you have `cabal-install` installed globally, it'll work
 inside the shell as expected.
 
-### shellFor {#ssec-haskell-shellFor}
+### shellFor {#sec-haskell-shellFor}
```

Comment:
Wondering if we should just drop the `sec-`/`ssec-` prefix altogether.

Reply:
I agree, since these anchors are semi user-facing.

Reply:
Okay, then let's do it. There are pre-existing anchors with both schemes in the manual, so it hardly matters.